docs/containers-registries.conf.5.md

```diff
@@ -109,6 +109,26 @@ internet without having to change `Dockerfile`s, or to add redundancy).
 *Note*: Redirection and mirrors are currently processed only when reading images, not when pushing
 to a registry; that may change in the future.
 
+#### Mirroring and rewriting the docker.io registry
+
+The default Docker registry `docker.io` is handled in a special way, because
+every push and pull operation gets internally suffixed with `/library`. This
```

Comment:
That's not entirely correct (see below). Not _every_ operation but only those missing a repository.